### ISSUE TITLE
I just lost 4 hours of note taking on this app.

### ISSUE THREAD

**Issue by bcnine** [Zettlr/Zettlr#4868]
### Description

I can't believe it. 4 solid hours of writing is lost on this app because its save functionality is broken. How the F can this happen?! I went back and did a test, and I was able to repro it. I would hit Ctrl+S to save my work, and it literally wouldn't save. I closed the app (stupidly) and re-opened it, and everything. Gone. Seriously folks, stop f'ing adding new features until you get your basic text editor 101 act together. This is garbage. Going back to sublime.

### Reproducing

Enable vim mode, type for a few hours, notice at some random point, Ctrl+S no longer saves your work. Voila. You've just wasted 4 hours.

### Zettlr Version

Stable (most recent version)

### Installation Method

None

### Your Platform

- [ ] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [ ] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 10

**Comment by nathanlesage** [comment]
Please mind your tone. Losing 4 hours of work is 100% frustrating, but please stay civil.

To reassure: you've experienced this bug in VIM mode, correct? Did you reproduce this bug in regular input mode?

**Comment by nathanlesage** [comment]
I will be closing this issue as I fear there won't be any additional response by OP. However, it is my interest to fix bugs, especially if they are this severe. Therefore, should you decide to give Zettlr a second chance, please comment here with more information and I will reopen this issue and begin tackling it asap.

**Comment by dstark** [comment]
> I will be closing this issue as I fear there won't be any additional response by OP. However, it is my interest to fix bugs, especially if they are this severe. Therefore, should you decide to give Zettlr a second chance, please comment here with more information and I will reopen this issue and begin tackling it asap.

@nathanlesage, I've been experiencing what seems to be this same issue in testing out Zettlr (Windows 10 and 11, Zettlr 3.0.5), though I'm using the "normal" edit mode (not VIM). Files newly created in Zettlr I can edit and save fine. I can still save things fine if I close and reopen Zettlr. But if I import a file (e.g., from DOCX or ODT), I routinely have to copy text out of those files and into a newly created file before I can save changes.

This has also cropped up once or twice around times when I've maybe closed, moved, and then readded workspaces. But I haven't been able to reproduce this portion reliably as yet.

Please just let me know if there's any debug output that would be helpful. Zettlr does some amazing things. It would be nice to not have to be concerned with watching whether a file had stopped saving properly.

**Comment by nathanlesage** [comment]
@dstark If it's reproducible specifically with *imported* files, that might warrant a new issue. Please ensure to test this problem on 3.1.1 so that we have the most recent code base as a reference for the bug.

**Comment by dstark** [comment]
> Please ensure to test this problem on 3.1.1

@nathanlesage, yes, my bad. I checked for updates before posting but had only installed Zettlr from PortableApps.com. Only after posting here did I realize that version hadn't yet updated to the more recent 3.1.1 being discussed elsewhere and now available for download from the main Zettlr site.

For me at least, the errors I'm encountering now appear wholly grouped under [this issue](https://github.com/Zettlr/Zettlr/issues/5109).

Thanks so much!
